native_client/javascript/Makefile

```diff
@@ -29,21 +32,28 @@ package.json: package.json.in
 		-e 's/$$(PROJECT_VERSION)/$(PROJECT_VERSION)/' \
 		package.json.in > package.json && cat package.json
 
-configure: deepspeech_wrap.cxx package.json
+npm-dev: package.json
+	# node-gyp@5.x behaves erratically with VS2015 and MSBuild.exe detection
```

Comment:
nit: move this inside of the `ifeq windows` so that it doesn't get printed every time this target runs.